app/src/main/java/com/vero/tasky/agenda/presentation/agendaevent/EventDetailsState.kt

```diff
@@ -1,12 +1,15 @@
 package com.vero.tasky.agenda.presentation.agendaevent
 
+import android.os.Parcelable
 import androidx.annotation.StringRes
 import com.vero.tasky.R
 import com.vero.tasky.agenda.domain.model.AgendaItem
+import com.vero.tasky.agenda.domain.model.Attendee
 import com.vero.tasky.agenda.presentation.model.ReminderRange
+import kotlinx.parcelize.Parcelize
 import java.time.LocalDateTime
 import java.util.*
-
+@Parcelize
```

Comment:
Why do you need the state to be parcelable?

Reply:
To save state in the SavedStateHandle

Reply:
Gotcha